### PR TITLE
Updating spire with latest images and CRDs

### PR DESCRIPTION
**Pull request by abhishek44sharma**

### CODE REVIEW

File: hack/init-next-helm-chart.sh [hunk 1]
```diff
@@ -7,7 +7,7 @@ helmChartDirName="helm-charts"
 filesToBeUpdated=("Chart.yaml" "charts/safe/Chart.yaml" "charts/sentinel/Chart.yaml" \
 "charts/spire/Chart.yaml")
 
-if [ "$1" == "-h" ]; then
+if [[ "$1" == "--h" || "$1" == "--help" ]]; then
```
Comment (farhan-pasha) [inline hack/init-next-helm-chart.sh:10]: I believe we should keep using one hyphen for one letter short options and two hyphens for long options
```suggestion
if [[ "$1" == "-h" || "$1" == "--help" ]]; then
```

Comment (abhishek44sharma) [reply hack/init-next-helm-chart.sh]: Agree, addressed!!

**Review by farhan-pasha — APPROVED** [review]
LGTM. Thanks

**Comment by v0lkan** [comment]
https://github.com/vmware-tanzu/secrets-manager/pull/362 has one.

I updated the CRDs to their correct versions, too.

Closing this one.

**Pull request closed without merging**